app/break.html

```diff
@@ -7,6 +7,7 @@
   </head>
   <body>
     <div class="microbreak">
+      <h2> Time to take a break! </h2>
```

Comment:
I think we do not need this, we can separate between break and microbreak very easy: 
- microbreak - one short sentence
- break - title and sentence

I prefer not to introduce this.